cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp

```diff
@@ -430,7 +430,7 @@ class NodeClass
 		bool srvCallback_Init(cob_srvs::Trigger::Request &req,
 							  cob_srvs::Trigger::Response &res )
 		{
-			ROS_DEBUG("Service Callback init");
+			ROS_DEBUG("Service callback /base_controller/init");
```

Comment:
@ipa-mig-tk Please take care of this. ipa-fxm has a point there. Didn't think of it when talking to you...
Just call it "Service init in cob_base_drive_chain" or something (same for below)